R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,96 @@
+# importCNA.GISTIC2.R
+#
+#
+#'@function Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#'@param fFHG Local fully qualified path of Firehose GISTIC2 CNA file.
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
+#'@param noLog Boolean. Default: FALSE. Whether or not to log results.
+#
+#
+#'@family importM.COSMIC, importM.TCGA, importM.GISTIC2, importM.ProjectGenie
+#
+#'@example
+# fFHG <- "inst/extdata/devCNA.txt"
+# dCNA <- "inst/extdata/dCNA"
+# silent <- FALSE
+# writeLog <- TRUE
+#
+# importCNA.GISTIC2(fFHG, dCNA, silent = FALSE, writeLog = TRUE)
+#
+# Input - all_data_by_genes.txt from the GISTIC2 output from FireHose
+# saved locally
+#
+# Output - rCNA file
+#
+#
+#
+
+importCNA.GISTIC2<- function(fFHG, dCNA, silent = FALSE, writeLog = TRUE){
+
+    # check if file exists
+
+    if(!file.exists(fFHG)){
+            stop(fFHG)
+    }
+
+    # parameter check
+
+    cR <- character()
+    cR <- c(cR, .checkArgs(fFHG,        like = "devCNA.txt",   checkSize = TRUE))
+    cR <- c(cR, .checkArgs(dCNA,        like = "inst/extdata/dCNA",
```

Comment:
This is misleading since it looks like a directory check but actually only checks for mode/typeof/class character. You can't check if the file exists, because it might not. But you could do:
```
.checkArgs(dirname(rCNA), like = "DIR", checkSize = TRUE)
```